<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>limits</class>
 <widget class="QDialog" name="limits">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="fiveVoltsConMax">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>4.500000000000000</double>
   </property>
   <property name="maximum">
    <double>6.500000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>5.500000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="fiveVoltsMax">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>4.500000000000000</double>
   </property>
   <property name="maximum">
    <double>6.500000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>5.500000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Naprázdno</string>
   </property>
  </widget>
  <widget class="QLabel" name="short_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>V zátěži</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Max</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Min</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Max</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Min</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>0</y>
     <width>16</width>
     <height>261</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>401</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>0</y>
     <width>16</width>
     <height>261</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="twelveVoltsMax">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>11.500000000000000</double>
   </property>
   <property name="maximum">
    <double>13.500000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>12.500000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="fifteenVoltsMax">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>14.500000000000000</double>
   </property>
   <property name="maximum">
    <double>16.500000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>15.500000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="ubatMax">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>9.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="twentyfourVoltsMax">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>15.000000000000000</double>
   </property>
   <property name="maximum">
    <double>30.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>24.500000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="ubatMin">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>8.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="fiveVoltsMin">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>3.500000000000000</double>
   </property>
   <property name="maximum">
    <double>5.500000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>4.500000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="fifteenVoltsMin">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>13.500000000000000</double>
   </property>
   <property name="maximum">
    <double>15.500000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>14.500000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="twelveVoltsMin">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>10.500000000000000</double>
   </property>
   <property name="maximum">
    <double>12.500000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>11.500000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="twentyfourVoltsMin">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>200</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>15.000000000000000</double>
   </property>
   <property name="maximum">
    <double>30.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>23.500000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="fiveVoltsConMin">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>3.500000000000000</double>
   </property>
   <property name="maximum">
    <double>5.500000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>4.500000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="ubatMaxShort">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>170</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>9.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="fiveVoltsMaxShort">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>4.500000000000000</double>
   </property>
   <property name="maximum">
    <double>6.500000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>5.500000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="fifteenVoltsMaxShort">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>140</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>14.500000000000000</double>
   </property>
   <property name="maximum">
    <double>16.500000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>15.500000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="twelveVoltsMaxShort">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>11.500000000000000</double>
   </property>
   <property name="maximum">
    <double>13.500000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>12.500000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="twentyfourVoltsMaxShort">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>200</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>15.000000000000000</double>
   </property>
   <property name="maximum">
    <double>30.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>24.500000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="fiveVoltsConMaxShort">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>50</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>4.500000000000000</double>
   </property>
   <property name="maximum">
    <double>6.500000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>5.500000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="ubatMinShort">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>170</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>8.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="fiveVoltsMinShort">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>80</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>3.500000000000000</double>
   </property>
   <property name="maximum">
    <double>5.500000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>4.500000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="fifteenVoltsMinShort">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>140</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>13.500000000000000</double>
   </property>
   <property name="maximum">
    <double>15.500000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>14.500000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="twelveVoltsMinShort">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>110</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>10.500000000000000</double>
   </property>
   <property name="maximum">
    <double>12.500000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>11.500000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="twentyfourVoltsMinShort">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>200</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>15.000000000000000</double>
   </property>
   <property name="maximum">
    <double>30.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>23.500000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="fiveVoltsConMinShort">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>3.500000000000000</double>
   </property>
   <property name="maximum">
    <double>5.500000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>4.500000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="oxyMin">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>15.000000000000000</double>
   </property>
   <property name="maximum">
    <double>30.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>23.500000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="oxyMinShort">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>230</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>15.000000000000000</double>
   </property>
   <property name="maximum">
    <double>30.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>23.500000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="oxyMaxShort">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>230</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>15.000000000000000</double>
   </property>
   <property name="maximum">
    <double>30.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>24.500000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="oxyMax">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>15.000000000000000</double>
   </property>
   <property name="maximum">
    <double>30.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>24.500000000000000</double>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>250</y>
     <width>401</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="fiveVoltsCon">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>5V_K</string>
   </property>
  </widget>
  <widget class="QLabel" name="fiveVolts">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>5V</string>
   </property>
  </widget>
  <widget class="QLabel" name="twelveVolts">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>12V</string>
   </property>
  </widget>
  <widget class="QLabel" name="fifteenVolts">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>15V</string>
   </property>
  </widget>
  <widget class="QLabel" name="U_bat">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ubat</string>
   </property>
  </widget>
  <widget class="QLabel" name="twentyfourVolts">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>24V</string>
   </property>
  </widget>
  <widget class="QLabel" name="twentyfourVolts_oxy">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>24V_O2</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>limits</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>limits</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
